src/components/Button/Button.scss

```diff
@@ -124,6 +124,11 @@
   border: none;
   vertical-align: top;
   line-height: normal;
+  font-size: 0;
+
+  > * + * {
```

Comment:
I'm sure this works, but I'd prefer we use class selectors to keep the code readable.